Solution1/ConsoleApp/Services/UserCommunicateService.cs

```diff
@@ -143,8 +144,14 @@ private async Task GetForecastByCityNameAsync()
                 _logger.LogError($"User entered incorrect value for 'countDay'.");
             }
 
-            var command = new ForecastWeatherCommand(_weatherServiсe, cityName, countDay);
-            var result = await _invoker.RunAsync(command, TokenGenerator.GetCancellationToken(_config.RequestTimeout));
+            var command = new ForecastWeatherCommand(
+                _weatherServiсe,
+                cityName,
+                countDay,
+                $"{_config.ApiConfig.ForecastWeatherUrl}{_config.ApiConfig.Key}",
+                $"{_config.ApiConfig.CoordinatesUrl}{_config.ApiConfig.Key}", _config.ApiConfig.CountPointsInDay);
```

Comment:
again and again

Reply:
if it will be necessary for example to put **-**  between params you will go throw 100500 places?